go/tools/gazelle/config/config.go

```diff
@@ -107,6 +144,12 @@ const (
 	// (declared in WORKSPACE).
 	ExternalMode DependencyMode = iota
 
+	// KnownMode indicates imports should be resolved via ExternalMode unless
```

Comment:
There doesn't need to be an extra mode of this. `KnownImports` can just be empty if there aren't any.